composer/datasets/imagenet.py

```diff
@@ -92,7 +92,7 @@ def initialize_object(self, batch_size: int, dataloader_hparams: DataloaderHpara
 class TinyImagenet200WebDatasetHparams(WebDatasetHparams):
     """Defines an instance of the TinyImagenet-200 WebDataset for image classification.
 
-    Parameters:
+    Args:
```

Comment:
References to `DataSpec` in the "Returns" section for the `initialize_object` method aren't resolving.